tests/integration_tests/test_message_queue_service.py

```diff
@@ -4,7 +4,9 @@
 import pytest
 
 from service import config
-from service.message_queue_service import MessageQueueService, message_to_dict, ConnectionAttemptFailed
```

Comment:
This is an incompatibility difference between the `black` and `isort`, maybe we'll be able to find a way to use both eventually :D